redbot/cogs/warnings/warnings.py

```diff
@@ -76,6 +84,49 @@ def __init__(self, bot: Red):
         else:
             await ctx.send(_("Custom reasons have been disabled."))
 
+    @warningset.command()
+    @commands.guild_only()
+    async def toggledm(self, ctx: commands.Context):
+        """Toggle whether warnings should be sent to users in DMs."""
+        guild = ctx.guild
+        toggle = not await self.config.guild(guild).toggle_dm()
+        await self.config.guild(guild).toggle_dm.set(toggle)
+        if toggle:
+            await ctx.send(_("Warnings will now be sent to users in DMs."))
```

Comment:
I know its not idea but we shouldnt say this in the affirmative, perhaps changing it to 
`"I will try to send warning to user in DMs."` since users can block the bot etc  its not garanteed warned users will get a warning
Same applied to the toggle off message